README.md

```diff
@@ -164,6 +165,10 @@ $ sha256sum -c --strict SHA256SUM.md
   slsa-verifier-linux-amd64: OK
 ```
 
+### Use Homebrew on macOS
+
+If you are using macOS and Homebrew, then you can install the verifier using this [formula](https://formulae.brew.sh/formula/slsa-verifier).
```

Comment:
Can you add you a note that the formula is maintained by the community.. or your GitHub handle?

Reply:
Done!